### ISSUE TITLE
Downloaded translations do not match current project configuration

### ISSUE THREAD

**Issue by zkrzyzanowski** [crowdin/crowdin-cli#14]
Running `crowdin download -b branch-name` and get the following output:
```Building ZIP archive with the latest translations - skipped
Warning: Export was skipped. Please note that this method can be invoked only once per 30 minutes.
Warning: Downloaded translations do not match current project configuration. The following files will be omitted:
 - '/app/locales/es/translations.json'
 - '/app/locales/fr/translations.json'
 - '/app/locales/it/translations.json'
 - '/app/locales/pt/translations.json'
```
crowdin.yaml:
```#
# Your crowdin's credentials
#
"project_identifier" : "impact-app"
"api_key_env" : CROWDIN_API_KEY
"base_path" : "."
#"base_url" : ""

#
# Choose file structure in crowdin
# e.g. true or false
#
#"preserve_hierarchy": true

#
# Files configuration
#
files: [
 {
  #
  # Source files filter
  # e.g. "/resources/en/*.json"
  #
  "source" : "/app/locales/en/translations.json",

  #
  # where translations live
  # e.g. "/resources/%two_letters_code%/%original_file_name%"
  #
  "translation" : "/app/locales/%two_letters_code%/%original_file_name%",

  #
  # files or directories for ignore
  # e.g. ["/**/?.txt", "/**/[0-9].txt", "/**/*\?*.txt"]
  #
  #"ignore" : [],

  #
  # The dest allows you to specify a file name on Crowdin
  # e.g. "/messages.json"
  #
  #"dest" : "",

  #
  # File type
  # e.g. "json"
  #
  #"type" : "",

  #
  # The parameter "update_option" is optional. If it is not set, translations for changed strings will be lost. Useful for typo fixes and minor changes in source strings.
  # e.g. "update_as_unapproved" or "update_without_changes"
  #
  #"update_option" : "",

  #
  # Start block only for XML
  #

  #
  # Defines whether to translate tags attributes.
  # e.g. 0 or 1  (Default is 1)
  #
  # "translate_attributes" : 1,

  #
  # Defines whether to translate texts placed inside the tags.
  # e.g. 0 or 1 (Default is 1)
  #
  # "translate_content" : 1,

  #
  # This is an array of strings, where each item is the XPaths to DOM element that should be imported
  # e.g. ["/content/text", "/content/text[@value]"]
  #
  # "translatable_elements" : [],

  #
  # Defines whether to split long texts into smaller text segments.
  # e.g. 0 or 1 (Default is 1)
  #
  # "content_segmentation" : 1,

  #
  # End block only for XML
  #

  #
  # Start .properties block
  #

  #
  # Defines whether single quote should be escaped by another single quote or backslash in exported translations.
  # e.g. 0 or 1 or 2 or 3 (Default is 3)
  # 0 - do not escape single quote;
  # 1 - escape single quote by another single quote;
  # 2 - escape single quote by backslash;
  # 3 - escape single quote by another single quote only in strings containing variables ( {0} ).
  #
  # "escape_quotes" : 3,

  #
  # End .properties block
  #

  #
  # Often software projects have custom names for locale directories. crowdin-cli allows you to map your own languages to be understandable by Crowdin.
  #
  #"languages_mapping" : {
  #  "two_letters_code" : {
  #    "crowdin_language_code" : "local_name"
  #   }
  #},

  #
  # Is first line contains header?
  # e.g. true or false
  #
  #"first_line_contains_header" : true,

  #
  # for spreadsheets
  # e.g. "identifier,source_phrase,context,uk,ru,fr"
  #
  # "scheme" : "",
 }
]
```
Using `v2.0.6`

**Comment by ihor-popyk** [comment]
@zkrzyzanowski, thank you for report. I'll try to reproduce and fix it soon.

**Comment by ihor-popyk** [comment]
@zkrzyzanowski, I fixed this bug in CLI 2.0.7 version. You can download it [here](http://crowdin.com/downloads/crowdin-cli.zip). Please, check it.
